### ISSUE TITLE
Sign in fails as "holo/agent/identify" method not found

### ISSUE THREAD

**Issue by alastairong** [Holo-Host/holo-envoy#64]
Browser Console error:
```
Unhandled Promise Rejection: HoloError: Identify Error: {"code":-32601,"message":"Method not found"}
```

Envoy logs:
```
Feb 15 16:27:43 hpos holo-envoy[749]: 2021-02-15T16:27:43.515Z [ index.js   ] SILLY: Incoming connection from /hosting/?anonymous=false&agent_id=uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc&hha_hash=uhCkkcF0X1dpwHFeIPI6-7rzM6ma9IgyiqD-othxgENSkL1So1Slt
Feb 15 16:27:43 hpos holo-envoy[749]: 2021-02-15T16:27:43.516Z [ index.js   ] NORMA: Agent (uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc) connection for HHA ID: uhCkkcF0X1dpwHFeIPI6-7rzM6ma9IgyiqD-othxgENSkL1So1Slt
Feb 15 16:27:43 hpos holo-envoy[749]: 2021-02-15T16:27:43.722Z [ index.js   ] SILLY: Incoming websocket message: {"jsonrpc":"2.0","method":"rpc.on","params":["signal"],"id":1}
Feb 15 16:27:43 hpos holo-envoy[749]: 2021-02-15T16:27:43.825Z [ index.js   ] SILLY: Incoming websocket message: {"jsonrpc":"2.0","method":"holo/wormhole/event","params":["uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc"],"id":2}
Feb 15 16:27:43 hpos holo-envoy[749]: 2021-02-15T16:27:43.826Z [ index.js   ] NORMA: Initializing wormhole setup for Agent (uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc)
Feb 15 16:27:43 hpos holo-envoy[749]: 2021-02-15T16:27:43.826Z [ index.js   ] DEBUG: Registering RPC WebSocket event (uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc/wormhole/request) in namespace: /hosting/
Feb 15 16:27:43 hpos holo-envoy[749]: 2021-02-15T16:27:43.826Z [ index.js   ]  WARN: RPC WebSocket event 'uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc/wormhole/request' is already registered for Agent (uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc)
Feb 15 16:27:44 hpos holo-envoy[749]: 2021-02-15T16:27:44.043Z [ index.js   ] SILLY: Incoming websocket message: {"jsonrpc":"2.0","method":"rpc.on","params":["uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc/wormhole/request"],"id":3}
Feb 15 16:27:44 hpos holo-envoy[749]: 2021-02-15T16:27:44.295Z [ index.js   ] SILLY: Incoming websocket message: {"jsonrpc":"2.0","method":"holo/agent/identify","params":["uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc"],"id":4}
Feb 15 16:27:44 hpos holo-envoy[749]: 2021-02-15T16:27:44.514Z [ index.js   ] NORMA: Socket is closing for Agent (uhCAkCQZSSO0xSbujgcpou0lIIGCDepyOSYYQOtoEhacH3I81OInc) using HHA ID uhCkkcF0X1dpwHFeIPI6-7rzM6ma9IgyiqD-othxgENSkL1So1Slt
```

**Comment by zo-el** [comment]
Looks like the chaperone is not been updated to sign-in correctly.
This should probably be its own defect story. 
I have started the work in this PR: https://github.com/Holo-Host/chaperone/pull/97

**Comment by zo-el** [comment]
This can be closed. it is Solved in the chaperone v1.1.0 release
